client/src/components/Workflow/Editor/test-data/parameter_steps.json

```diff
@@ -943,8 +945,8 @@
       "top": 867.0427551269531
     }
   },
-  "19": {
-    "id": 19,
+  "20": {
+    "id": 20,
```

Comment:
Should this be backported to release_24.1, as it was introduced in https://github.com/galaxyproject/galaxy/pull/18810/files#diff-bb81ba432ebe3591ccddfd36ab261ec668b57170ca10ed9c39acf727e2a17cbd ?
It doesn't seem to be a problem there, but also not correct.